packages/react/src/MinimalFooter/MinimalFooter.test.tsx

```diff
@@ -0,0 +1,478 @@
+import {render} from '@testing-library/react'
+import '@testing-library/jest-dom'
+import {axe, toHaveNoViolations} from 'jest-axe'
+import {MinimalFooter} from './MinimalFooter'
+import {Text} from '../'
+
+expect.extend(toHaveNoViolations)
+
+// Helper function to filter social links
+const getSocialLinks = (links: HTMLElement[]) => {
+  return links.filter(
+    link =>
+      link.getAttribute('href')?.includes('x.com') ||
+      link.getAttribute('href')?.includes('github.com/github') ||
+      link.getAttribute('href')?.includes('linkedin.com') ||
+      link.getAttribute('href')?.includes('youtube.com') ||
+      link.getAttribute('href')?.includes('facebook.com') ||
+      link.getAttribute('href')?.includes('twitch.tv') ||
+      link.getAttribute('href')?.includes('tiktok.com') ||
+      link.getAttribute('href')?.includes('instagram.com'),
+  )
+}
+
+describe('MinimalFooter', () => {
+  it('has no accessibility violations', async () => {
+    const {container} = render(<MinimalFooter />)
+    const results = await axe(container)
+    expect(results).toHaveNoViolations()
+  })
+
+  it('renders as a footer element', () => {
+    const {getByRole} = render(<MinimalFooter />)
+    expect(getByRole('contentinfo')).toBeInTheDocument()
+  })
+
+  it('renders default copyright statement', () => {
+    const {getByText} = render(<MinimalFooter />)
+    const currentYear = new Date().getFullYear()
+    expect(getByText(`© ${currentYear} GitHub. All rights reserved.`)).toBeInTheDocument()
+  })
+
+  it('renders custom copyright statement', () => {
+    const customCopyright = 'Custom Copyright 2024'
+    const {getByText} = render(<MinimalFooter copyrightStatement={customCopyright} />)
+    expect(getByText(customCopyright)).toBeInTheDocument()
+  })
+
+  it('renders copyright statement as React element', () => {
+    const customCopyright = <span>Custom Copyright 2024</span>
+    const {getByText} = render(<MinimalFooter copyrightStatement={customCopyright} />)
+    expect(getByText('Custom Copyright 2024')).toBeInTheDocument()
+  })
+
+  it('renders GitHub logo with default href', () => {
+    const {getByRole} = render(<MinimalFooter />)
+    const logoLink = getByRole('link', {name: 'GitHub'})
+    expect(logoLink).toHaveAttribute('href', 'https://github.com')
+  })
+
+  it('renders GitHub logo with custom href', () => {
+    const customHref = 'https://custom.github.com'
+    const {getByRole} = render(<MinimalFooter logoHref={customHref} />)
+    const logoLink = getByRole('link', {name: 'GitHub'})
+    expect(logoLink).toHaveAttribute('href', customHref)
+  })
+
+  it('applies custom className', () => {
+    const {getByRole} = render(<MinimalFooter className="custom-footer" />)
+    expect(getByRole('contentinfo')).toHaveClass('custom-footer')
+  })
+
+  it('renders all social links by default', () => {
+    const {getAllByRole} = render(<MinimalFooter />)
+    const socialLinks = getSocialLinks(getAllByRole('link'))
```

Comment:
This feels a little hard to read in isolation because the core test logic is abstracted away into getSocialLinks. Related comment here. Can we instead have individual assertions like you did for tests below. It's more verbose, but easier to reason about IMO

Reply:
I found a nicer way to do this which leverages [Testing Library's TextMatch function](https://testing-library.com/docs/queries/about/#textmatch), which essentially lets Testing Library figure it out, lmk what you think